RS/2D vector space/main.py

```diff
@@ -0,0 +1,137 @@
+import pygame, sys, time, random, math
+from pygame.locals import *
+
+pygame.init()
+pygame.mixer.init()
+
+# Declare all variables etc
+BLACK = (0, 0, 0)
+WHITE = (255, 255, 255)
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+BLUE = (0, 0, 255)
+
+windowWidth = 256
+windowHeight = 256
+
+tailLength = 20
+lumTotal = 0
+lumCount = 0
+doingSomethingToPassTheTime = 0
+firstPart = True
+secondPart = False
+
+window = pygame.display.set_mode((windowWidth, windowHeight))
+picture = pygame.image.load('pic1.jpg')
```

Comment:
Why `pic1.jpg`? You could make this a bit more dynamic with `'pic' + PICTURE_TO_LOAD + '.jpg'` and `PICTURE_TO_LOAD = 1`, etc.